scripts/hostmap-crtsh.nse

```diff
@@ -88,31 +96,52 @@ local function is_valid_hostname (name)
   return true
 end
 
+-- TRUE subdomain check (ends with ".parent")
+local function is_subdomain(name, parent)
+  name = name:lower()
+  parent = parent:lower()
+  local suffix = "." .. parent
```

Comment:
Also prudent, but expensive. Consider that that for every name in every certificate you are repeatedly converting to lowercase (and prepending) the same `hostname` string parameter given to `query_ctlogs()`. You would be better off just to calculate `suffix` only once, at the beginning of `query_ctlogs()`, and pass it to `is_subdomain()` instead of `parent`.